src/views/layout/template.njk

```diff
@@ -19,10 +19,16 @@
 {% endblock %}
 
 {% block header %}
-  {{ govukHeader({
-    homepageUrl: homePageUrl,
-    serviceName: serviceName
-  }) }}
+  <header class="header-container">
```

Comment:
Maybe we can restructure to improve readability and render responsively for small devices. 
However, we do not have visual designs for different screens. But we get responsiveness with the DEFRA template and styles. But with the custom header, the UI is not the same for small devices.

```
<header role="banner">
  <div class="defra-header">
    <div class="defra-header__logo">
      <a class="defra-header__link">
        <img />
        <span class="defra-header__product-name"></span>
      <a>
    </div>
    <div class="defra-header__content">
      <span class="defra-header__service-name"></span>
    </div>
  </div>
</header>
```

- I believe we need to include a link on the image to navigate to the defra organization page.
- By having the content, we can push it below the logo block on small screens, which is similar to the approach we get from the DEFRA default header.

Reply:
we have to let it be fluid as we don't have designs for the header responsiveness. however we already requested the design team on it.